topics/statistics/tutorials/FNN/slides.html

```diff
@@ -0,0 +1,321 @@
+	- As many neurons in output layer as number of classes
+	- Sigmoid activation function
+	- Activation > 0.5, output 1
+	- Activation <= 0.5, output 0
+---
+
+# Output layer (Continued)
+
+- Multiclass classification
+	- As many neurons in output layer as number of classes
+	- *Softmax* activation function
+	- Takes input to neurons in output layer 
+	- Creates a probability distribution, sum of outputs adds up to 1 
+	- The neuron with the highest proability is the predicted label
+- Regression problem
+	- Sinfle neuron in output layer
+	- Linear activation function
+
+---
+
+# Loss/Cost functions
+
+- During training, for each training example (x^i,y^i), we present x^i to neural network
+	- Compare predicted output with label y^1 
+	- Need loss function to measure difference between predicted & expected output
+- Use *Cross entropy* loss function for classification problems 
+- And *Quadratic* loss function for regression problems
+	- Qadratic cost function is also called *Mean Squared Error (MSE)* 
+
+---
+
+# Cross Entropy Loss/Cost functions
+
+![Cross Entropy loss function]({{site.baseurl}}/topics/statistics/images/FNN_Cross_Entropy_Loss.png) <!-- https://pixy.org/3013900/ CC0 license-->
+
+![Cross Entropy cost function]({{site.baseurl}}/topics/statistics/images/FNN_Cross_Entropy_Cost.png) <!-- https://pixy.org/3013900/ CC0 license-->
+
+---
+
+# Quadratic Loss/Cost functions
+
+![Quadratic loss function]({{site.baseurl}}/topics/statistics/images/FNN_Quadratic_Loss.png) <!-- https://pixy.org/3013900/ CC0 license-->
+
+![Quadratic cost function]({{site.baseurl}}/topics/statistics/images/FNN_Quadratic_Cost.png) <!-- https://pixy.org/3013900/ CC0 license-->
+
+---
+
+# Backpropagation (BP) learning algorithm
+
+- A *gradient descent* technique
+	- Find local minimum of a function by iteratively moving in opposite direction of gradient of function at current point 
+- Goal of learning is to minimize cost function given training set 
+	- Cost function is a function of network weights & biases of all neurons in all layers 
+	- Backpropagation iteratively computes gradient of cost function relative to each weight and bias
+	- Updates weights and biases in the opposite direction of gradient 
+	- To find a local minimum
+
+---
+
+# Backpropagation error
+
+![Backpropagation error]({{site.baseurl}}/topics/statistics/images/FNN_BP_Error.png) <!-- https://pixy.org/3013900/ CC0 license-->
+
+---
+
+# Backpropagation formulas
+
+![Backpropagation formulas]({{site.baseurl}}/topics/statistics/images/FNN_BP.png) <!-- https://pixy.org/3013900/ CC0 license-->
+
+---
+
+# Types of Gradient Descent
+
+- *Batch* gradient descent
+	- Calculate gradient for each weight/bias for *all* samples
+	- Average gradients and update weights/biases
+	- Slow, if we have too many samples  
+
+- *Stochastic* gradient descent
+	- Update weights/biases based on gradient of *each* sample
+        - Fast. Not accurate if sample gradient not representiative
```

Comment:
Is the indentation correct for `- Fast`?

Reply:
No, fixed it. Thanks.